admin/test/test_installer.py

```diff
@@ -387,49 +386,6 @@ def _cleanup_flocker(self):
         )
         return d
 
-    def _cleanup_compose(self):
-        """
-        Run docker-compose stop and rm -f for both demo templates to stop and
-        remove all the containers that were created during the test.
-        Run serially because docker-compose + swarm sometimes fail when
-        commands are run in parallel.
-        """
-        d_node1_compose = remote_docker_compose(
-            self.client_node_ip,
-            self.docker_host,
-            self.compose_node1.path, 'stop'
-        )
-        d_node1_compose.addCallback(
-            lambda ignored: remote_docker_compose(
-                self.client_node_ip,
-                self.docker_host,
-                self.compose_node1.path, 'rm', '-f'
-            ).addErrback(
-                # This sometimes fails with exit code 255
-                # and a message ValueError: No JSON object could be decoded
-                lambda failure: failure.trap(ProcessTerminated)
-            )
-        )
-        d_node2_compose = remote_docker_compose(
-            self.client_node_ip,
-            self.docker_host,
-            self.compose_node2.path,
-            'stop',
-        )
-        d_node2_compose.addCallback(
-            lambda ignored: remote_docker_compose(
-                self.client_node_ip,
-                self.docker_host,
-                self.compose_node2.path,
-                'rm', '-f'
-            ).addErrback(
-                # This sometimes fails with exit code 255
-                # and a message ValueError: No JSON object could be decoded
-                lambda failure: failure.trap(ProcessTerminated)
-            )
-        )
-        return gather_deferreds([d_node1_compose, d_node2_compose])
```

Comment:
`docker-compose down` stops and removes the containers related to the compose file so these commands are no longer needed.